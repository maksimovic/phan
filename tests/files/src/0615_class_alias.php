<?php

\class_alias('stdClass', 'a_615');
