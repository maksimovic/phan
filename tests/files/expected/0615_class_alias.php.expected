%s:3 PhanParamMustBeUserDefinedClassname First argument of class_alias() must be a name of user defined class ('%s' attempted)
